### PR TITLE
Ccfri 3655 mtfi remove fees question / get MTFI in working order / general cleanup efforts

### PR DESCRIPTION
**Pull request by jenbeckett**
# Description
MTFI RFI is still broken - but once the other ticket I have for RFI is in - I feel like that should more or less be fixed. 

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/mtfi/CurrentFeeVerification.vue [hunk 1]
```diff
@@ -1264,15 +840,8 @@ export default {
     hasModelChanged() {
       return !isEqual(this.CCFRIFacilityModel, this.loadedModel);
     },
-    arePrevFeesCorrect() {
-      return !(
-        !this.CCFRIFacilityModel.existingFeesCorrect || this.CCFRIFacilityModel.existingFeesCorrect == '100000001'
-      );
-    },
     isInputVisible(index) {
```
Comment (vietle-cgi) [inline frontend/src/components/mtfi/CurrentFeeVerification.vue:843]: Could this one be simplified to: "return !isEmpty(this.CCFRIFacilityModel.childCareTypes[index].feeFrequency);"?

File: frontend/src/components/summary/changeRequest/MTFISummary.vue [hunk 2]
```diff
@@ -41,229 +41,201 @@
...
+                      flat
+                      variant="solo"
+                      hide-details
+                      readonly
+                      prefix="$"
+                    />
+                  </div>
+
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    July
```
Comment (vietle-cgi) [inline frontend/src/components/summary/changeRequest/MTFISummary.vue:87]: should this one be "Jul"?

File: frontend/src/components/summary/changeRequest/MTFISummary.vue [hunk 3]
```diff
@@ -273,229 +245,201 @@
...
+                      flat
+                      variant="solo"
+                      hide-details
+                      readonly
+                      prefix="$"
+                    />
+                  </div>
+
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    Jen
```
Comment (vietle-cgi) [inline frontend/src/components/summary/changeRequest/MTFISummary.vue:291]: I love to have a Jen's month but maybe we should use "Jan"? :D

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/components/mtfi/CurrentFeeVerification.vue [hunk 4]
```diff
@@ -971,23 +631,6 @@ import { isEqual } from 'lodash';
 import ApiService from '../../common/apiService.js';
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/components/mtfi/CurrentFeeVerification.vue:631]: yup! done

File: frontend/src/components/summary/changeRequest/MTFISummary.vue (hunk 2 above)
Comment (jenbeckett) [reply frontend/src/components/summary/changeRequest/MTFISummary.vue:87]: done

File: frontend/src/components/summary/changeRequest/MTFISummary.vue (hunk 3 above)
Comment (jenbeckett) [reply frontend/src/components/summary/changeRequest/MTFISummary.vue:291]: Probably ahahah

File: frontend/src/components/mtfi/CurrentFeeVerification.vue (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/components/mtfi/CurrentFeeVerification.vue:843]: The facility model always comes loaded with a selection so the drop down works.. So your code change would always return true

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**